src/MetalsFeatures.ts

```diff
@@ -8,6 +8,8 @@ export interface TreeViewProvider {}
 
 export class MetalsFeatures implements StaticFeature {
   treeViewProvider?: TreeViewProvider;
+  supportsDebugging?: Boolean;
```

Comment:
For consistency with capability providers in LSP
```suggestion
  debuggingProvider?: DebuggingProvider;
```

This can be an empty object for now

```ts
interface DebuggingProvider {}
```

The benefit of this approach is that we can add more fields in the future to configure what kind of debugging functionality is supported.